examples/js/CSVnVRTParser.js

```diff
@@ -0,0 +1,207 @@
+/* global itowns, Promise, THREE */
+
+/**
+ * A module to parse OGR Virtual Format files.
+ *
+ * See the [GDAL documentation](https://gdal.org/drivers/vector/vrt.html) and
+ * the [xsd
+ * schema](https://github.com/OSGeo/gdal/blob/master/gdal/data/ogrvrt.xsd) of
+ * the OGR VRT file.
+ *
+ * @example
+ * Fetcher.multi('data', {
+ *     xml: ['vrt'],
+ *     text: ['csv']
+ * }).then(function _(res) {
+ *     return CSVnVRTParser.parse(res, {
+ *         buildExtent: true,
+ *         crsOut: 'EPSG:4326'
+ *     });
+ * }).then(function _(feature) {
+ *     var source = new itowns.FileSource({ parsedData: feature });
+ *     var layer = new itowns.ColorLayer('CSVnVRT', { source });
+ *     view.addLayer(layer);
+ * });
+ *
+ * @module CSVnVRTParser
+ */
+var CSVnVRTParser = (function _() {
+    var coord = new itowns.Coordinates('EPSG:4326');
+
+    function csv2json(csv) {
```

Comment:
may be you could use existing packages to convert `csv2json` and `xml2json`, to lighten the code. WDYT?

Reply:
I agree that this may be better, as there is a lot of things that can be taking into account to parse a CSV (more than I thought of). However I can't find something working correctly for now, I'll try to do my best about that.